### DISCUSSION TITLE
build issue

### DISCUSSION THREAD

**Discussion by ChunHuCui** [wormhole-foundation/wormhole#551] [Problems / bugs]
Hi, development team.
I have got an error in running this command.
npm ci --prefix sdk/js
![image](https://user-images.githubusercontent.com/68866354/138344340-a98e2f53-9829-416a-915b-4998d6e46c76.png)

thank you in advance.

**Comment by evan-gray** [comment]
Hi! Looks like you are missing some files from our repo. Try cloning again? That folder should look like https://github.com/certusone/wormhole/tree/dev.v2/sdk/js. Then follow the instructions in https://github.com/certusone/wormhole/blob/dev.v2/bridge_ui/README.md

**Comment by ChunHuCui** [comment]
I already checked out it dev.v2, but I don't know why my path is not a tree/dev.v2..
![image](https://user-images.githubusercontent.com/68866354/138723311-f672c608-3fc7-4f9e-a593-f86fdfeb0d11.png)

**Comment by evan-gray** [comment]
Could you please verify all of the steps in `bridge_ui/README.md` and share the output? 
```
docker --version
node --version
npm --verison
DOCKER_BUILDKIT=1 docker build --target node-export -f Dockerfile.proto -o type=local,dest=. .
DOCKER_BUILDKIT=1 docker build -f solana/Dockerfile.wasm -o type=local,dest=. solana
npm ci --prefix ethereum
npm ci --prefix sdk/js
npm run build --prefix sdk/js
```
